src/Nio/OutMgr.cpp

```diff
@@ -125,6 +125,17 @@ string OutMgr::getSink() const
     return "ERROR";
 }
 
+void OutMgr::setAudioCompressor(bool isEnabled)
+{
+    printf("OutMgr - setting isOutputCompressionEnabled to: %d\n", isEnabled);
```

Comment:
Good for development, but this should be removed for the final merging.

Reply:
ok